config.json

```
@@ -485,6 +485,21 @@
                 "math",
                 "test_driven_development"
             ]
-        }
+        },
+        {
+      "slug": "nth-prime",
+      "uuid": "7ce09989-f202-4c3c-8b7e-72cef18808c3",
+      "core": false,
+      "unlocked_by": "robot-name",
```

```suggestion
"unlocked_by": null,
```

I'll have to think about where this fits into the progression, let's add it as floating exercise for now.